include/runtime/instance/memory.h

```diff
@@ -294,16 +320,11 @@ class MemoryInstance {
   /// \param Length the store length to data. Need to <= sizeof(T).
   ///
   /// \returns void when success, ErrCode when failed.
-  template <typename T>
+  template <typename T, uint32_t const Length = sizeof(T)>
   typename std::enable_if_t<IsWasmNativeNumV<T>, Expect<void>>
-  storeValue(const T &Value, uint32_t Offset, uint32_t Length) noexcept {
+  storeValue(const T &Value, uint32_t Offset) noexcept {
     // Check the data boundary.
-    if (unlikely(Length > sizeof(T))) {
-      spdlog::error(ErrCode::MemoryOutOfBounds);
-      spdlog::error(
-          ErrInfo::InfoBoundary(Offset, Length, Offset + sizeof(T) - 1));
-      return Unexpect(ErrCode::MemoryOutOfBounds);
-    }
+    static_assert(Length <= sizeof(T));
```

Comment:
As above.

Reply:
Same reason to use `static_assert` as above.

Reply:
As explained above.